test/test_pipelines.py

```diff
@@ -22,3 +23,26 @@ def test_interface_in_blocks():
             gr.Interface.from_pipeline(pipe2)
     demo.launch(prevent_thread_lock=True)
     demo.close()
+
+
+def test_transformers_load_from_pipeline():
+    from transformers import pipeline
+
+    pipe = pipeline(model="deepset/roberta-base-squad2")
+    io = Interface.from_pipeline(pipe)
+    assert io.input_components[0].label == "Context"
+    assert io.input_components[1].label == "Question"
+    assert io.output_components[0].label == "Answer"
+    assert io.output_components[1].label == "Score"
+
+
+def test_diffusers_load_from_pipeline():
+    from diffusers import StableDiffusionPipeline
+
+    pipe = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5")
```

Comment:
Does this mean the test will download the weights for SD1.5? If so, that should be avoided.

Reply:
I have removed this test for now.